src/material/radio/radio.ts

```diff
@@ -646,14 +676,15 @@ export class MatRadioButton implements OnInit, AfterViewInit, DoCheck, OnDestroy
   }
 
   _onInputClick(event: Event) {
-    // We have to stop propagation for click events on the visual hidden input element.
-    // By default, when a user clicks on a label element, a generated click event will be
-    // dispatched on the associated input element. Since we are using a label element as our
-    // root container, the click event on the `radio-button` will be executed twice.
-    // The real click event will bubble up, and the generated click event also tries to bubble up.
-    // This will lead to multiple click events.
-    // Preventing bubbling for the second event will solve that issue.
-    event.stopPropagation();
```

Comment:
I removed this because it turned out that before this change we were never actually calling the `_onInputClick` callback. It must've gotten lost during the MDC transition.